rr.go

```diff
@@ -50,6 +50,11 @@ func convertRR(drr dns.RR) (RR, bool) {
 		return RR{toLowerFQDN(t.Hdr.Name), "AAAA", t.AAAA.String()}, true
 	case *dns.TXT:
 		return RR{toLowerFQDN(t.Hdr.Name), "TXT", strings.Join(t.Txt, "\t")}, true
+	default:
```

Comment:
The comment on line 38 should reflect the new behavior of this func.

Reply:
function description updated

as for your question regarding which records this would work on, to be honest you could probably replace the entire function with the default. as it works for just about all records (since the reply is just about always in this format). however since your lowercasing the output of dns records, and i didn't want to lowercase the entire output, I left the original request types as is.

so i've added the function as is, to ensure that atleast MX, PTR, CAA, HINFO, MINFO, MB, MG, MR, WKS, requests work. i added tests for the once i could publicly use (MX/PRT). but the rest will work too.